tools/tokenserver/loadtests/locustfile.py

```diff
@@ -19,50 +21,79 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         # Keep track of this user's generation number.
         self.generation_counter = 0
-        self.x_key_id = self._make_x_key_id_header()
+        self.client_state = binascii.hexlify(
+            self.generation_counter.to_bytes(16, 'big')).decode('utf8')
         # Locust spawns a new instance of this class for each user. Using the
         # object ID as the FxA UID guarantees uniqueness.
         self.fxa_uid = id(self)
-        self.email = "loadtest-%s@%s" % (self.fxa_uid, MOCKMYID_DOMAIN)
+        self.email = "loadtest-%s@%s" % (self.fxa_uid, FAKE_DOMAIN)
 
-    @task(1000)
-    def test_success(self):
+    @task(3000)
+    def test_oauth_success(self):
         token = self._make_oauth_token(self.email)
 
-        self._do_token_exchange(token)
+        self._do_token_exchange_via_oauth(token)
+
+    @task(100)
+    def test_invalid_oauth(self):
+        token = self._make_oauth_token(status=400)
 
-    @task(5)
-    def test_invalid_scope(self):
+        self._do_token_exchange_via_oauth(token, status=401)
+
+    @task(100)
+    def test_invalid_oauth_scope(self):
         token = self._make_oauth_token(
             user=str(self.fxa_uid),
             scope=["unrelated", "scopes"],
         )
 
-        self._do_token_exchange(token, status=401)
-
-    @task(5)
-    def test_invalid_token(self):
-        token = self._make_oauth_token(status=400, errno=108)
+        self._do_token_exchange_via_oauth(token, status=401)
 
-        self._do_token_exchange(token, status=401)
-
-    @task(5)
+    @task(20)
     def test_encryption_key_change(self):
         # When a user's encryption keys change, the generation number and
         # keys_changed_at for the user both increase.
         self.generation_counter += 1
-        self.x_key_id = self._make_x_key_id_header()
+        self.client_state = binascii.hexlify(
+            self.generation_counter.to_bytes(16, 'big')).decode('utf8')
         token = self._make_oauth_token(self.email)
 
-        self._do_token_exchange(token)
+        self._do_token_exchange_via_oauth(token)
 
-    @task(5)
+    @task(20)
     def test_password_change(self):
         # When a user's password changes, the generation number increases.
         self.generation_counter += 1
         token = self._make_oauth_token(self.email)
 
-        self._do_token_exchange(token)
+        self._do_token_exchange_via_oauth(token)
+
+    @task(100)
+    def test_browserid_success(self):
+        assertion = self._make_browserid_assertion(self.email)
```

Comment:
I'm having trouble confirming this in the old Python code but aren't browserid assertions encoded somehow (vs just being a json dumps here)?

Or is this currently intended to solely work with the mock verification service (and would need tweaking to work with say a live stage env)?